UlKit/UlKit/OrderViewController.swift

```diff
@@ -0,0 +1,148 @@
+        return button
+    }()
+
+    private var leftPatternImageView = UIImageView(image: .leftPattern)
+
+    private var rightPatternImageView = UIImageView(image: .rightPattern)
+
+    private var orderLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Вашъ Заказъ"
+        label.font = .verdanaBold18()
+        label.textColor = .black
+        return label
+    }()
+
+    private var finalCostLabel: UILabel = {
+        let label = UILabel()
+        label.font = .verdanaBold18()
+        label.textColor = .black
+        return label
+    }()
+
+    private var patternLogoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "menu_pic")
+        return imageView
+    }()
+
+    private lazy var payButton: CustomButton = {
+        let button = CustomButton(type: .system)
+        button.setTitle("Оплатить", for: .normal)
+        button.titleLabel?.font = .verdanaBold18()
+        button.setTitleColor(.white, for: .normal)
+        button.frame.origin = CGPoint(x: 15, y: 600)
+        button.addTarget(self, action: #selector(goToNextViewController), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addSubviews()
+        setupLabels()
+    }
+
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        setupViews()
+    }
+
+    // MARK: - Private Methods
+
+    private func addSubviews() {
+        view.backgroundColor = .white
+        view.addSubview(leftPatternImageView)
+        view.addSubview(closeButton)
+        view.addSubview(rightPatternImageView)
+        view.addSubview(orderLabel)
+        view.addSubview(finalCostLabel)
+        view.addSubview(patternLogoImageView)
+        view.addSubview(payButton)
+    }
+
+    private func setupLabels() {
+        if let additions = orderItems {
+            for (index, (ingredient, price)) in additions.enumerated() {
+                let priceLabel = UILabel(frame: CGRect(x: offsetXSubview, y: offsetYSubview, width: labelWidth, height: labelHeight))
+                let ingredientLabel = UILabel(frame: CGRect(
+                    x: offsetXSubview,
+                    y: offsetYSubview,
+                    width: labelWidth,
+                    height: labelHeight
+                ))
+                ingredientLabel.text = "\(ingredient)"
+                priceLabel.text = "\(price) руб"
+                priceLabel.textAlignment = .right
+
+                if index == 0 {
+                    ingredientLabel.font = .verdanaBold16()
+                    priceLabel.font = .verdanaBold16()
+                } else {
+                    ingredientLabel.font = .verdana16()
+                    priceLabel.font = .verdana16()
+                }
+
+                labels.append(ingredientLabel)
+                offsetYSubview += labelHeight + 10
```

Comment:
```suggestion
                offsetYSubview += labelHeight + 
```
Магические числа в константы